pkgs/top-level/all-packages.nix

```diff
@@ -6766,6 +6766,8 @@ with pkgs;
       lldb_20 = llvmPackages_20.lldb;
       llvm_20 = llvmPackages_20.llvm;
       bolt_20 = llvmPackages_20.bolt;
+
+      llvmMkPackages = llvmPackagesSet.mkPackage;
```

Comment:
I think it should be called `mkLLVMPackages`

Reply:
Updated. I agree it's better that way.